pages/oku/_location.vue

```diff
@@ -10,26 +10,58 @@
 <script>
 import OKU from '~components/oku-circ'
 import moment from 'moment'
+import { mapState } from 'vuex'
 
 export default {
   data () {
     return {
-      okuLocation: this.$route.params.location,
-      currentTime: moment().format('MMM D / h:mm A')
+      okuLocation: this.$route.params.location
     }
   },
   components: {
     'oku-circ': OKU
   },
+  computed: {
+    ...mapState({
+      currentTime: state => moment(state.time.now).format('MMM D / h[<span class="blink">:</span>]mm A')
+    })
+  },
   async fetch ({ store, params }) {
     await store.dispatch('laptops/fetchStatus', 'olin')
     await store.dispatch('phoneChargers/fetchStatus', 'olin')
     await store.dispatch('laptops/fetchStatus', 'uris')
     await store.dispatch('phoneChargers/fetchStatus', 'uris')
   },
-  layout: 'oku'
+  layout: 'oku',
+  mounted () {
+    // Sync current time every 10 seconds
+    // -- ideally, this would happen within the store itself (as part of the action),
+    // -- but encountered a bug when attempting this via Nuxt. No issue in vanilla Vue.
+    setInterval(() => {
+      this.$store.dispatch('time/sync')
+    }, 1000 * 10)
+
+    // Update equipment status every minute
+    setInterval(() => {
+      this.$store.dispatch('laptops/fetchStatus', 'olin')
+      this.$store.dispatch('phoneChargers/fetchStatus', 'olin')
+      this.$store.dispatch('laptops/fetchStatus', 'uris')
+      this.$store.dispatch('phoneChargers/fetchStatus', 'uris')
+    }, 1000 * 60)
+  }
 }
 </script>
 
-<style>
+<style lang="scss">
```

Comment:
I would add support for the rest of the browsers. 
I like to add first the css rule, them the animation. I see this pattern often. 
Also, I like to use the shorthand version. Just my preference.

Once I notice the blinker, my eyes focus just on that :)

.blink {
    -webkit-animation: blinker infinite cubic-bezier(1.0,0,0,1.0) 1s;
    animation: blinker infinite cubic-bezier(1.0,0,0,1.0) 1s;
  }

  @-webkit-keyframes blinker {
    from { opacity: 1.0; }
    to { opacity: 0.0; }
  }

  @keyframes blinker {
    from { opacity: 1.0; }
    to { opacity: 0.0; }
  }